### ISSUE TITLE
0.35.0 build failure

### ISSUE THREAD

**Issue by ilovezfs** [edporras/pdftoedn#4]
```
==> autoreconf -i
configure.ac:33: installing './compile'
configure.ac:85: installing './config.guess'
configure.ac:85: installing './config.sub'
configure.ac:11: installing './install-sh'
configure.ac:11: installing './missing'
src/Makefile.am: installing './depcomp'
==> ./configure --with-openssl=/usr/local/opt/openssl --prefix=/usr/local/Cellar/pdftoedn/0.35.0
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable debugging... no
./configure: line 3396: syntax error near unexpected token `-Wno-user-defined-warnings,'
./configure: line 3396: `AX_CHECK_COMPILE_FLAG(-Wno-user-defined-warnings, CXXFLAGS="$CXXFLAGS -Wno-user-defined-warnings")'
```

**Comment by edporras** [comment]
This is likely due to the formula missing a build dependency for [autoconf-archive](https://github.com/Homebrew/homebrew-core/blob/master/Formula/autoconf-archive.rb):

`depends_on "autoconf-archive" => :build`

Would you like me to submit an updated `pdftoedn` formula?

**Comment by edporras** [comment]
Oops, apologies. I see I didn't copy the required m4 files. Pushing a commit shortly.

**Comment by edporras** [comment]
Included files and pushed a 0.35.1 release. Sorry about that.
